website/src/app/kb/deploy/resources/readme.mdx

```diff
@@ -46,6 +46,19 @@ From there, you can select the type of Resource you want to create:
   Resource.
 </Alert>
 
+#### Routing order for overlapping addresses
+
+When multiple Resources' addresses overlap, the Resource with the more specific
+address will be used. For CIDR Resources, this means the Resource with the
+longest prefix length will be used.
```

Comment:
Why is the "For CIDR Resources" part of this paragraph but for DNS, we start a new one?

Reply:
Hm I thought it was too choppy to split it up, but no strong feelings.